.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,235 @@
+name: Pipeline
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'feature**'
+  delete:
+    branches:
+      - 'feature**'
+
+env:
+  SAM_TEMPLATE: template.yaml
+  TESTING_STACK_NAME: pdf-encrypt-dev
+  TESTING_PIPELINE_EXECUTION_ROLE: arn:aws:iam::415725062401:role/aws-sam-cli-managed-dev-pipel-PipelineExecutionRole-SQp73by0rIkQ
```

Comment:
Never ever leave secrets like this in the open 

Basically anyone can come in and use your ARN right?